docs-ref-conceptual/microsoft-graph-migration.md

```diff
@@ -0,0 +1,133 @@
+
+### `az ad app permission grant`
+
+- `--expires` argument is removed
+
+### `az ad app credential reset`
+
+- `--credential-description` is replaced by `--display-name` (https://github.com/Azure/azure-cli/issues/20561)
+- `--password` is removed. Without specifying certificate arguments, Graph service creates a password for you (https://github.com/Azure/azure-cli/issues/20675)
+
+### `az ad sp delete`
+
+- This command no longer deletes the corresponding application. Use `az ad app delete` to explicitly delete the application (https://github.com/Azure/azure-cli/issues/8467)
+- This command no longer deletes corresponding role assignments of the service principal. Use `az role assignment delete` to explicitly delete role assignments (https://github.com/Azure/azure-cli/issues/20805)
+
+### `az ad sp credential`
+
+- This command group now operates on service principal, not application (https://github.com/Azure/azure-cli/issues/11458)
+
+### `az ad sp credential reset`
+
+- `--password` is removed. Without specifying certificate arguments, Graph service creates a password for you (https://github.com/Azure/azure-cli/issues/20675)
+
+### `az ad group get-member-groups`
+
+- `--additional-properties` argument is removed
+
+### `az ad group member add`
+
+- `--additional-properties` argument is removed
+
+## Known issues
+
+- Generic update arguments `--add`, `--set` and `--remove` currently don't work
+- `az ad` and Microsoft Graph related commands like `az role` will fail in Azure Stack environments which don't have Microsoft Graph support
+
+## Try Azure CLI beta with Microsoft Graph
+
+Azure CLI is built on [Python](https://www.python.org/). The supported Python versions are 3.6, 3.7, 3.8. On Windows, you will first need to [install Python](https://www.python.org/downloads/windows/).
+
+Azure CLI beta can only be installed with `pip` from a Microsoft repository. Use `python` or `python3` to execute the following commands, depending on the Linux distribution or your installed Python version.
+
+To avoid overwriting your installed Azure CLI, we recommend installing the beta version in a [virtual environment](https://docs.python.org/3/tutorial/venv.html).
+
+1. Create a virtual environment
+
+   Navigate to the folder where you want to create the virtual environment, then run:
+
+   ```bash
+   python -m venv <env_name>
+   ```
+
+1. Activate the virtual environment
+
+   ### [Windows PowerShell](#tab/powershell)
+
+   ```powershell
+   . .\<env_name>\Scripts\Activate.ps1
+   ```
+
+   ### [Linux/macOS Bash](#tab/bash)
+
+   ```bash
+   . <env_name>/bin/activate
+   ```
+   ---
+   These commands can also be used to reactivate your virtual environment.
+
+1. Install Azure CLI beta
+
+   ```bash
+   python -m pip install --upgrade pip
+   pip install --extra-index-url https://azcliprod.blob.core.windows.net/beta/simple/ azure-cli
```

Comment:
```suggestion
   pip install --extra-index-url https://azcliprod.blob.core.windows.net/beta/simple/azure-cli
```

I don't think there is supposed to be a space here? Correct me if I'm wrong. it will install but with a warning and adjusted url:
```Output
warning: missing-index-doctype

× The package index page being used does not have a proper HTML doctype declaration.
╰─> Problematic URL: https://azcliprod.blob.core.windows.net/beta/simple/azure-cli/
```

Reply:
No, the space is required because `azure-cli` is the package name. 

The page doesn't work because we haven't published beta yet. I will do that today.